src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -131,7 +134,7 @@ export class MdAutocompleteTrigger implements AfterContentInit, ControlValueAcce
    * A stream of actions that should close the autocomplete panel, including
    * when an option is selected and when the backdrop is clicked.
    */
-  get panelClosingActions(): Observable<any> {
+  get panelClosingActions(): Observable<MdOptionSelectEvent | null> {
```

Comment:
What's your reasoning for doing `| null` here since we haven't been doing explicit null yet?